modules/codegen/src/main/scala/com/twilio/guardrail/generators/CirceProtocolGenerator.scala

```diff
@@ -141,6 +142,7 @@ object CirceProtocolGenerator {
             case s: StringSchema    => EmptyValueIsNull(s)
             case _                  => None
           }).getOrElse(EmptyIsEmpty)
+          dataRedaction = DataRedaction(property).getOrElse(DataVisible)
```

Comment:
I think this feature will eventually need to be controllable via core interpreter, defining the default based on call site (possibly plugin configuration) such that for secure environments everything should be redacted by default

Reply:
Yeah, that makes sense.  Do you think we'd want to parse it out here, and then just give the core interp a chance to override it?

Reply:
Yeah -- I'm thinking a single `Term` that just `Target.pure(DataVisible)` defining the default. Eventually, it may consume the `config` parameters and use plugin configuration to differentiate, but I haven't thought that far ahead, mainly just interested in separating static defaults.